packages/composites/linear-finance/test/integration/__snapshots__/adapter.test.ts.snap

```diff
@@ -81,13 +67,6 @@ Object {
           },
         },
       },
-      "CELO": Object {
```

Comment:
Not sure if this is intentional but these values are not returned by the API but were previously present in the CSV files.  Are they supposed to be time dependent? @boxhock

Reply:
They were probably in a previous version of the index, I'm guessing